dirty_cat/_gap_encoder.py

```diff
@@ -303,20 +303,6 @@ def fit(self, X, y=None) -> "GapEncoderColumn":
         self.H_dict_.update(zip(unq_X, unq_H))
         return self
 
-    def get_feature_names(self, n_labels=3, prefix=""):
-        """
-        Ensures compatibility with sklearn < 1.0.
-        Use `get_feature_names_out` instead.
-        """
-        warnings.warn(
-            "Following the changes in scikit-learn 1.0, "
-            "get_feature_names is deprecated. "
-            "Use get_feature_names_out instead. ",
-            DeprecationWarning,
-            stacklevel=2,
-        )
-        return self.get_feature_names_out(n_labels=n_labels, prefix=prefix)
-
```

Comment:
We still support earlier versions of sklearn where ``get_feature_names`` is used (<1.0), so we should keep this method for now :)

Reply:
We can discuss in #436 but I think this function is unused. The function used for this is at line 850/836.

Reply:
Ok, to be removed then when get_feature_names is no longer used.